### PR TITLE
Fixup wasm loading in the browser

### PR DESCRIPTION
**Pull request by mischnic**
This time, I've actually tested the browser version...

Also adds the web build to the release workflow and builds + tests it in the test workflow

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -39,3 +41,9 @@ jobs:
         run: yarn run build:node-release
       - name: Run node tests
         run: yarn run test:node
+      - name: Build wasm release
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.node-version == '14.x' }}
+        run: yarn run build:wasm-node-release
+      - name: Run wasm tests
+        if: ${{ matrix.os == 'ubuntu-latest' && matrix.node-version == '14.x' }}
+        run: yarn run test:wasm
```
[the remark this answers is not in the document]
Comment (DeMoorJasper) [reply .github/workflows/test.yml:49]: Yeah maybe a good idea

File: .github/workflows/release.yml [hunk 2]
```diff
@@ -183,6 +183,8 @@ jobs:
     steps:
       - uses: actions/checkout@v1
       - uses: bahmutov/npm-install@v1.1.0
+      - name: Install wasm-pack
+        run: curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
```
Comment (DeMoorJasper) [inline .github/workflows/release.yml:187]: Could also use the node module https://yarnpkg.com/package/wasm-pack

Comment (mischnic) [reply .github/workflows/release.yml:187]: I tried that, and it didn't work (failed with `Not installed: "wasm-pack"` at runtime)

Comment (mischnic) [reply .github/workflows/release.yml:187]: https://github.com/parcel-bundler/source-map/runs/2659683400?check_suite_focus=true

Comment (DeMoorJasper) [reply .github/workflows/release.yml:187]: Oh that's strange, as long as it works it's probably fine

**Review by DeMoorJasper — APPROVED** [review]

**Pull request merged**